_data/whats-new.yml

```diff
@@ -4,21 +4,93 @@ description: |
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: 2018-12-20 14:21:59 -0600
+updated: 2019-01-14 14:11:39 -0600
 entries:
+- description: Updated the [availability timeline](https://devdocs.magento.com/availability)
+    for the Amazon Sales Channel extension.
+  versions: 'N/A'
+  type: Major update
+  date: January 10, 2019
+  link: https://github.com/magento/devdocs/pull/3552
+- description: "Added new topics to Integrations Testing Framework annotations
+    describing:<br/>- [@magentoAppArea](https://devdocs.magento.com/guides/v2.3/test/integration/annotations/magento-app-area.html)<br/>-
+    [@magentoAppIsolation](https://devdocs.magento.com/guides/v2.3/test/integration/annotations/magento-app-isolation.html)<br/>-
+    [@magentoCache](https://devdocs.magento.com/guides/v2.3/test/integration/annotations/magento-cache.html)<br/>-
+    [@magentoComponentsDir](https://devdocs.magento.com/guides/v2.3/test/integration/annotations/magento-components-dir.html)<br/>-
+    [@magentoConfigFixture](https://devdocs.magento.com/guides/v2.3/test/integration/annotations/magento-config-fixture.html)<br/>-
+    [@magentoDataFixture](https://devdocs.magento.com/guides/v2.3/test/integration/annotations/magento-data-fixture.html)<br/>-
+    [@magentoDbIsolation](https://devdocs.magento.com/guides/v2.3/test/integration/annotations/magento-db-isolation.html)"
+  versions: 2.1.x, 2.2.x, 2.3.x
+  type: New topic
+  date: January 10, 2019
+  link: https://github.com/magento/devdocs/pull/2665
+- description: Rewrite the [developer installation](https://devdocs.magento.com/guides/v2.3/install-gde/prereq/dev_install.html)
```

Comment:
"Rewrote" instead of "Rewrite" to be consistent with tense

Reply:
The change must be implemented in the corresponding PR

Reply:
PR #3512

Reply:
updated the related PR